### PR TITLE
Memory loss after death pt 2, because manatee doesnt know how to maintain

### PR DESCRIPTION
**Pull request by solwashere**
# Document the changes in your pull request

redone because manatee is dumb

Makes it so that if someone is dead for (currently) 5 minutes, upon revival a message will play telling the player that they have memory loss, either 5 or 10 minutes would likely be good, I'll let council decide that.
Message is tied to brain decay, so switching brains wouldn't work, and it should let you preserve the brain in order to preserve someone's "memories".

If someone is revived with memory loss special flavor text will play stating so.

Cloning text is now large, and logs

IPCs will now completely lose memory upon death

This will need a council vote.

# Why this is good for the game
This makes hiding targets a lot more viable, especially witnesses, while not removing the risk entirely of someone yelling your name. In theory this will mean that less people will get round removed if they don't need to be

# Wiki Documentation

Mention on the IPC wiki page that they suffer memory loss upon death
# Changelog

:cl:  

tweak: being dead for a while will make your memory of said death go bye bye 
tweak: IPCs will lose memory upon death
tweak: Clone memory loss text is large and red now
tweak: Cloning logs now are a thing

/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
its that easy...

**Comment by Therandomhoboo** [comment]
IPC/Pretenis buff

**Comment by neeshacark** [comment]
why do ipcs entirely lose their memory? not only does it complicate this but it makes ipc even weaker

**Comment by Marmio64** [comment]
> why do ipcs entirely lose their memory? not only does it complicate this but it makes ipc even weaker

likely because of IPC's unique brains not decaying and because this PR would tie memory loss to brain decay. And it doesn't really make them 'weaker', it mostly would just stop them from spilling the beans on an antag who killed them.

File: code/modules/hydroponics/grown/replicapod.dm [hunk 1]
```diff
@@ -120,7 +120,7 @@
 		podman.set_cloned_appearance()
 		podman.dna.species.exotic_blood = max(reagents_add) || /datum/reagent/water
 		log_cloning("[key_name(mind)] cloned as a podman via [src] in [parent] at [AREACOORD(parent)].")
-		to_chat(podman, span_notice("You do not remember your death, how you died, or who killed you. <a href='https://forums.yogstation.net/help/rules/#rule-1_6'>See rule 1.6</a>.")) //yogs
+		to_chat(podman, span_userdanger("You do not remember your death, how you died, or who killed you. <a href='https://forums.yogstation.net/help/rules/#rule-1_6'>See rule 1.6</a>.")) //yogs
```
Comment (adamsong) [inline code/modules/hydroponics/grown/replicapod.dm:123]: No combat log

File: code/modules/mob/living/brain/brain_item.dm [hunk 2]
```diff
@@ -214,8 +216,10 @@
 		owner.death()
 		brain_death = TRUE
 
-/obj/item/organ/brain/process()	//needs to run in life AND death
+/obj/item/organ/brain/process(delta_time)	//needs to run in life AND death
 	..()
+	if(!((organ_flags & ORGAN_SYNTHETIC) || (owner && (owner.stat < DEAD || HAS_TRAIT(owner, TRAIT_PRESERVED_ORGANS)))))
```
Comment (adamsong) [inline code/modules/mob/living/brain/brain_item.dm]: This ignores stasis entirely

**Review by adamsong — APPROVED** [review]

**Review by adamsong — CHANGES_REQUESTED** [review]
See previous comment

**Comment by solwashere** [comment]
> See previous comment

I'll look at it when I'm off work

**Comment by solwashere** [comment]
> > 
why do ipcs entirely lose their memory? not only does it complicate this but it makes ipc even weaker
> 
> likely because of IPC's unique brains not decaying and because this PR would tie memory loss to brain decay. And it doesn't really make them 'weaker', it mostly would just stop them from spilling the beans on an antag who killed them.

Too real
People are acting like this is a "buff" or "nerf" when it is too complex of an issue to qualify it as either. At most it'd probably be annoying for IPCs

**Comment by neeshacark** [comment]
in practice it adds another thing that makes them less of a viable choice over any other race.  they have very few benefits and every pr seems to make them weaker

**Comment by solwashere** [comment]
> in practice it adds another thing that makes them less of a viable choice over any other race.  they have very few benefits and every pr seems to make them weaker

"Less viable choice" I'm fairly certain this is honestly better as it makes it so that you can reliably kill an IPC and not have to RR them, or at least you don't have an excuse to do so

**Review by ToasterBiome — APPROVED** [review]

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
combat log + adam changes

**Comment by Marmio64** [comment]
> in practice it adds another thing that makes them less of a viable choice over any other race. they have very few benefits and every pr seems to make them weaker

ipcs just got 3 separate buff PRs in the past 2 weeks

**Comment by solwashere** [comment]
done

**Review by adamsong — APPROVED** [review]

**Review by adamsong — CHANGES_REQUESTED** [review]
Fix build errors

**Comment by solwashere** [comment]
Changed to SHOULD_LIFETICK due to https://github.com/yogstation13/Yogstation/pull/19341

**Review by adamsong — APPROVED** [review]
lgtm

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**